sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -841,6 +825,26 @@ class CodegenContext {
     }
   }
 
+  def splitCodes(expressions: Seq[String]): Seq[String] = {
```

Comment:
actually `split` is not accurate here, how about `groupCodes`?

Reply:
nit: private. We also need to write a comment.

How about `buildCodeBlocks`?

Reply:
Sure, but it is not `private`.  It is used in `stringExpressions`.